css/src/components/checkbox.scss

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-border-color: $text !default;
+$checkbox-color: $primary !default;
+$checkbox-check-color: $primary-invert !default;
+$checkbox-check-hover-color: $text-subtle !default;
+$checkbox-size: 1em !default;
+$checkbox-icon-size: 0.8em !default;
+$checkbox-timing-function: $input-timing-function !default;
+$checkbox-duration: $input-transition-duration !default;
+$checkbox-spacing: $spacer-3;
+
+.checkbox {
+	display: flex;
+	position: relative;
+	align-items: center;
+	font-size: $font-size-5;
+	line-height: 1.25;
+	cursor: pointer;
+
+	+ .checkbox {
```

Comment:
Should we remove the `.checkbox + .checkbox` rule entirely? This type of spacing is what atomics are for.

Reply:
removed